style.css

```diff
@@ -0,0 +1,586 @@
+  --fw-reg: 300;
+  --fw-bold: 900;
+  --clr-light: #eaeaea;
+  --clr-dark: #272a31;
+  --clr-accent: #db372a;
+  --clr-light2: #eaeaeac2;
+  --fs-h1: 3rem;
+  --fs-h2: 2.25rem;
+  --fs-h3: 1.25rem;
+  --fs-body: 1rem;
+  --fa-style-family-classic: "Font Awesome 6 Free";
+  --fa-font-solid: normal 900 1em/1 "Font Awesome 6 Free";
+  --bs:
+    0.25em 0.25em 0.75em rgba(0, 0, 0, 0.25),
+    0.125em 0.125em 0.25em rgba(0, 0, 0, 0.15);
+
+  scroll-behavior: smooth;
+}
+
+@font-face {
+  font-family: 'Lato', sans-serif;
+  src: url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap');
+}
+
+.btn {
+  display: inline-block;
+  padding: 0.5em 2.5em;
+  cursor: pointer;
+  font-size: 0.8rem;
+  letter-spacing: 2px;
+  font-weight: var(--fw-bold);
+  transition: transform 200ms ease-in-out;
+}
+
+.btn:hover {
+  transform: scale(1.1);
+  background-color: var(--clr-light2);
+  color: var(--clr-accent);
+  text-decoration: none;
+}
+
+.nav__link:hover,
+.social__link:hover {
+  color: var(--clr-accent);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: var(--ff-primary);
+  line-height: 1.6;
+}
+
+.nav-toggle {
+  font-size: 1.5rem;
+  color: var(--clr-dark);
+  width: 100vw;
+  padding: 1rem 1.25rem;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main-intro {
+  background: var(--clr-light) url('./assets/logos/logo-30anniv-2c-uai-516x470.png') no-repeat top;
+  height: 100%;
+  width: 100%;
+  background-blend-mode: overlay;
+  padding: 30px 10px;
+}
+
+.intro-greeting,
+.intro-title {
+  color: var(--clr-accent);
+}
+
+.intro-greeting {
+  font-size: 1.5rem;
+}
+
+.intro-title {
+  font-weight: var(--fw-bold);
+  font-size: var(--fs-h1);
+  line-height: normal;
+  margin: 0;
+}
+
+.intro-description {
+  border: 3px solid white;
+  background-color: var(--clr-light2);
+}
+
+.main-events {
+  background: var(--clr-dark) url('./assets/hitmar-resized.png');
+  color: var(--clr-light);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  padding: 2rem 1rem;
+  gap: 10px;
+}
+
+.events-cards {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.events-card {
+  display: grid;
+  grid-template-columns: 1fr 2fr 3fr;
+  align-items: center;
+  padding: 0.5rem;
+  background-color: rgba(120, 121, 133, 0.199);
+  gap: 0.5rem;
+}
+
+.more-info {
+  color: var(--clr-light);
+  background-color: var(--clr-accent);
+  border-radius: 0.25rem;
+  text-decoration: none;
+  text-align: center;
+  padding: 1rem;
+  width: 250px;
+  margin-top: 20px;
+}
+
+.events-icon {
+  width: 5rem;
+}
+
+.events-title,
+.featured-title {
+  margin-bottom: 0;
+}
+
+.events__card-title {
+  color: var(--clr-accent);
+  text-align: center;
+}
+
+hr {
+  width: 15%;
+  border: 1px solid var(--clr-accent);
+}
+
+.featured-artist {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  width: 100%;
+  padding: 2rem 1rem;
+}
```

Comment:
- Hey, for the desktop version, let's increase the padding to 3rem 9rem to improve our design to be close to the design template and to improve the user experience on the desktop version.




*design*
![Screenshot from 2023-04-20 00-12-16](https://user-images.githubusercontent.com/59397159/233219774-7012ac63-84c3-4028-ac02-2333eff01569.png)

*your design*
![Screenshot from 2023-04-20 00-10-52](https://user-images.githubusercontent.com/59397159/233219777-f69c7236-90c4-4858-bae7-3f9d3ec33a61.png)